consensus/src/liveness/leader_reputation.rs

```diff
@@ -322,55 +322,6 @@ impl NewBlockEventAggregation {
     }
 }
 
-/// If candidate appear in the history, it's assigned active_weight otherwise inactive weight.
```

Comment:
Can you update the PR description to reflect this change as well with reasoning behind it?